### PR TITLE
[ANB] extra cards

### PR DESCRIPTION
**Pull request by tiera3**
Baloth Packhunter
Compound Fracture
Hallowed Priest
Mardu Outrider
Tin Street Cadet

### CODE REVIEW

File: Mage.Sets/src/mage/cards/m/MarduOutrider.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package mage.cards.m;
+
+import mage.MageInt;
+import mage.abilities.Ability; // not sure about this line - think it is not needed
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/m/MarduOutrider.java:4]: fyi - Unused imports are harmless, though I certainly recommend just having your IDE manage that sort of thing automatically

File: Mage.Sets/src/mage/cards/t/TinStreetCadet.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    public TinStreetCadet(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{R}");
+
+        this.subtype.add(SubType.GOBLIN);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(1);
+
+        // Whenever Tin Street Cadet becomes blocked, create a 1/1 red Goblin creature token.
+        this.addAbility(new BecomesBlockedByCreatureTriggeredAbility(new CreateTokenEffect(new GoblinToken()), false));
```
Comment (xenohedron) [inline Mage.Sets/src/mage/cards/t/TinStreetCadet.java:28]: this one should be `BecomesBlockedSourceTriggeredAbility`

**Review by xenohedron — CHANGES_REQUESTED** [review]
"becomes blocked" is subtly different from "becomes blocked by a creature", so need to use the right class for it.

Rest looks good.

File: Mage.Sets/src/mage/cards/c/CompoundFracture.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+/**
+ * @author tiera3 - modified from Growth Cycle
+ */
+public final class CompoundFracture extends CardImpl {
+
+    private static final FilterCard filter = new FilterCard("card named Compound Fracture");
+    static {
+        filter.add(new NamePredicate("Compound Fracture"));
+    }
+    private static final DynamicValue xValue = new CardsInControllerGraveyardCount(filter, -1);
```
Comment (JayDi85) [inline Mage.Sets/src/mage/cards/c/CompoundFracture.java:25]: For info, not pr. If you see xValue then it’s a good point to add some card hint about calculated value for a user. Current example: dynamic card hint with “com fra cards in your graveyard: 123”

**Review by JayDi85 — APPROVED** [review]

**Pull request merged**